README.md

```diff
@@ -15,6 +15,10 @@ One intention is that all new official tools begin life in this repository. And
 over time we'll develop some criteria for promoting a user tool to an official
 tool that bpftrace maintainers will help support over long periods of time.
 
+## Timer tools
```

Comment:
Sorry, didn't notice this before. This blurb should be moved here:
https://github.com/bpftrace/user-tools/blob/0c310c2c834e83314f1d3136eae2d29e9d1efdee/TOOLS.md